src/App.js

```diff
@@ -1,24 +1,22 @@
-import logo from './logo.svg';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+import Books from './components/books';
+import Categories from './components/categories';
+import Nav from './components/nav';
```

Comment:
- [ ] Kindly add a component that creates books. It should contain a form with inputs for the title and the author of the book. It should also contain a button to submit which would just be a placeholder for now.